openstack/nova/templates/hypervisors/hypervisors-vmware-deployment-vct.yaml

```diff
@@ -103,8 +108,17 @@ template: |
           command: ["dumb-init"]
           args: ["neutron-dvs-agent", "--config-file", "/etc/neutron/neutron.conf", "--config-file", "/etc/neutron/plugins/ml2/ml2-conf.ini", "--config-file", "/etc/neutron/plugins/ml2/ml2-vmware.ini"]
           env:
-          - name: SENTRY_DSN
-            value: {{ .Values.neutron.sentry_dsn | quote }}
+            {{- if .Values.sentry.enabled }}
```

Comment:
@fwiesel If you check for capabilities (which nobody else does btw.), you'd have to check here, too.

Reply:
Ah, I was planning to remove that part (See #471)

Reply:
Exactly for things like this, I think it is sensible to move the dvs agent out of the nova chart.
We need to copy values around and keep them in sync.